packages/web-vite/src/components/LibraryItemCard.tsx

```
@@ -182,12 +189,29 @@ const LibraryItemCard: React.FC<LibraryItemCardProps> = ({
 
       {/* Dropdown menu - positioned outside thumbnail to avoid overflow clipping */}
       {showMenu && !isMultiSelectMode && showThumbnail && (
-        <div ref={menuRef} className="card-menu-dropdown" onClick={(e) => e.stopPropagation()}>
+        <div
+          ref={menuRef}
+          className="card-menu-dropdown"
+          onClick={(e) => e.stopPropagation()}
+        >
           <button
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)